tokio-console/src/view/mod.rs

```diff
@@ -180,32 +191,46 @@ impl View {
         update_kind
     }
 
+    /// The help modal should toggle on the `?` key and should exit on `Esc`
+    fn should_toggle_help_modal(&mut self, event: &crossterm::event::Event) -> bool {
+        input::is_help_toggle(event) || (self.show_help_modal && input::is_esc(event))
+    }
+
     pub(crate) fn render<B: ratatui::backend::Backend>(
         &mut self,
         frame: &mut ratatui::terminal::Frame<B>,
         area: layout::Rect,
         state: &mut State,
     ) {
-        match self.state {
+        let help_text: &dyn HelpText = match self.state {
```

Comment:
nit, take it or leave it: I'm not 100% convinced that the `HelpText` trait is *really* necessary here...couldn't we just have the `help_text` variable be a `Paragraph`, and have each view type have an inherent `help_text()` method that returns a `Paragraph`?

Reply:
i guess the advantage of this approach is that we can assign `help_text` in the same `match` expression as rendering the view without having to actually allocate to build the paragraph if we're not displaying the help text. but, we could alternatively just put an additional `match` in the `if self.show_help_modal` block, which would call the current state's method to get a `Paragraph` of help text...

it's up to you whether this is actually better, i think the trait is *fine*, i just don't know if we really need it.